frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -164,6 +165,28 @@ const plugin: Plugin<ConsumedExtensions> = [
       query: `${CAPACITY_USAGE_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED]}[60m:5m]`,
     },
   },
+  {
+    type: 'Dashboards/Storage/TopConsumers/Item',
+    properties: {
+      name: 'VMs',
+      queryNamePrefix: 'VMS_',
```

Comment:
this is an internal implementation and should not be needed to specify in extension definition. I think you could just reuse `name` prop.